File: sdk/src/main/java/org/coralibre/android/sdk/internal/util/ByteToHex.java
package org.coralibre.android.sdk.internal.util;

public class ByteToHex {
    public static String toString(byte[] data) {
        final StringBuilder builder = new StringBuilder();
        for (byte b : data) {
            builder.append(String.format("%02X", b));
        }
        return builder.toString();
    }
}

File: sdk/src/main/java/org/coralibre/android/sdk/internal/util/ByteToHex.kt
package org.coralibre.android.sdk.internal.util

object ByteToHex {
    @JvmStatic
    fun toString(data: ByteArray): String {
        val builder = StringBuilder()
        for (b in data) {
            builder.append(String.format("%02X", b))
        }
        return builder.toString()
    }
}
